regression/CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ add_subdirectory(goto-harness)
 add_subdirectory(goto-cc-file-local)
 add_subdirectory(linking-goto-binaries)
 add_subdirectory(symtab2gb)
+add_subdirectory(goto-analyzer-get-fp-values)
```

Comment:
This change needs to be reflected in the `regression/Makefile` I believe to get the tests to run on Make builds

Reply:
Done.

Weird this wasn't manifesting before. I just broke up the first commit, so there is no code changes - only different commits.

By the way, the github view for commit history is borked. The commits are alright but they show differently than in the command line git log.